Add `junit5-migration-maven-kotlin` sample #105 (#107)
junit-team/junit5-samples: README.md

The [junit5-migration-maven] sample demonstrates how to set up a Maven project
using the JUnit Platform, JUnit Jupiter, and JUnit Vintage.

### Maven & Kotlin Migration ![badge-jdk-8] ![badge-tool-maven] ![badge-junit-jupiter] ![badge-junit-vintage]

The [junit5-migration-maven-kotlin] sample demonstrates how to execute tests written in Java and Kotlin

> based on JUnit 5 using Maven. In addition, it showcases that existing JUnit 4 based tests can be executed

Reviewer: ```suggestion
using JUnit 5 with Maven. Additionally, it demonstrates that existing JUnit 4 tests can also be executed
```
